src/main/java/Main.java

```diff
@@ -0,0 +1,37 @@
+import java.io.IOException;
+
+import duke.Duke;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke("data/duke.ser").usingMainUi();
+
+    /**
+     * MainWindow loaded from MainWindow.
+     * DialogBox initialised from DialogBox.java instead of loaded from fxml.
+     */
+    @Override
+    public void start(Stage stage) {
+        assert stage != null;
```

Comment:
Give a description/comment for your assert to aid troubleshooting

Reply:
I guess it differs from person to person. But IMO, it should be self-explanatory enough that I wouldn't think of placing any comment.

Reply:
Hello, I think I was not clear in my initial comment. What I meant was that when you assert, perhaps you can do `assert stage != null : "Stage not found"; ` as that will make it clear what the problem is when your assert fails.

Reply:
😄 I was wondering if you meant an assert message, which is great. But I concluded you suggest me to leave a comment by the assert.